deep-learning/_posts/2017-04-11-sharpmask.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: review
+title:  "Feature Pyramid Networks for Object Detection"
+tags:   deep-learning CNN localization bounding-boxes
+author: Frédéric Branchaud-Charron
+pdf:   "https://arxiv.org/pdf/1612.03144.pdf"
+cite:
+  authors: "T. Lin, P. Dollár, R. Girshick, K. He, B. Hariharan, S. Belongie"
+  title:   "Feature Pyramid Networks for Object Detection"
+  venue:   "Computer Vision and Pattern Recognition, 2017. CVPR 2017"
+---
+
+SharpMask was designed as a successor to DeepMask. The main drawbacks from DeepMask were speed and performances.
```

Comment:
maybe mention that as for DeepMask, SharpMask can be used as an object detection method when combined with R-CNN